test/token/ERC20/extensions/ERC20Votes.test.js

```diff
@@ -347,60 +347,11 @@ contract('ERC20Votes', function (accounts) {
       });
     });
 
-    describe('numCheckpoints', function () {
```

Comment:
We need to keep all tests. This shows why: this PR should be a refactor, and it turns out we're accidentally (I hadn't realized) removing the functions `numCheckpoints` and `checkpoints`. If we want to remove these functions, it's a separate discussion that we haven't had, so we need to add them back.